src/tree/SubscriptionTreeItem.ts

```diff
@@ -88,4 +92,19 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     public isAncestorOfImpl(contextValue: string): boolean {
         return contextValue !== AttachedStorageAccountTreeItem.baseContextValue && contextValue !== AttachedStorageAccountTreeItem.emulatedContextValue;
     }
+
+    private async getDefaultLocation(wizardContext: IStorageAccountWizardContext): Promise<string> {
+        const stackLocation: SubscriptionModels.Location | undefined = (await LocationListStep.getLocations(wizardContext)).find(l => l.displayName !== undefined || l.name !== undefined);
+        let defaultStackLocation: string;
```

Comment:
nit: Since this isn't always the stack location
```suggestion
        let defaultLocation: string;
```